components/automate-cli/pkg/verifyserver/utils/httputils/httputil.go

```diff
@@ -0,0 +1,33 @@
+package httputils
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
+)
+
+//Post - This function performs HTTP Post request to the given endpoint with the provided request body
+func MakePostRequest(url string, body interface{}) (*http.Response, error) {
```

Comment:
Make its name to MakeRequest. Which can support all the type of request. You can accept methodType, request body. In case of get request you can pass nil